openpype/plugins/publish/extract_review.py

```diff
@@ -713,6 +714,11 @@ def _ffmpeg_arguments(
             ffmpeg_output_args.extend(audio_out_args)
 
         res_filters = self.rescaling_filters(temp_data, output_def, new_repre)
+
+        # Override Scaling filters if source resolution is set as tag
+        if "source_resolution" in output_def['tags']:
+            res_filters = [f for f in res_filters if "scale" not in f]
```

Comment:
Doesn't it make more sense to just skip the rescaling filters completely instead of 'filtering it out afterwards'?

Reply:
Hey thanks for checking, yes i don't see why not skipping all rescaling filters. Just wanted to remove the 'scale' one in our case.